portfolio/src/main/webapp/script.js

```diff
@@ -28,11 +28,23 @@ function addRandomGreeting() {
   greetingContainer.innerText = greeting;
 }
 
-function getHelloMessage(){
+// Fetches a JSON with messages from the server, creates elements with the messages and adds them to the DOM:
+function getMessages(){
 
-    fetch("/data").then(response => response.text()).then((HelloMessage) => {
+    fetch("/data").then(response => response.json()).then((messages) => {
 
-        document.getElementById("message-zone").innerHTML = HelloMessage;
+        //Iterates the JSON by keys:
+        for(var key in messages){
```

Comment:
use `let` instead of `var` to better communicate to readers when a variable stays the same with `const` or changes with `let`

Reply:
I never clearly understood the difference between `let` and `var` until now. Thanks. I will change it.